### PR TITLE
Rename grantee to recipient

### PR DESCRIPTION
**Pull request by jasalisbury**
## Description of change

Update grantee to recipient on the frontend. Grantee TTA record page hasn't been released, so routes and variables can be changed without regard to compatibility between different versions of the FE and BE. Other places we aren't so lucky. A separate change which will take much longer will be coming down the pipe to address the grantee -> recipient change fully in the codebase, including updating routes/the database.

## How to test

This branch is currently deployed in dev.

View all pages and fill out reports. Make sure fields are labeled correctly and everything still works. Be sure to download a CSV and verify fields are not labeled with `grantee`. Values inside the form have not been updated nor has any data, both will be updated in a later PR.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-443

### Every PR

- [x] Meets issue criteria
- [x] JIRA ticket status updated
- [x] Code is meaningfully tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

**Review by AdamAdHocTeam — APPROVED** [review]
Looks good. Just need to make sure the Recipient nav bar label matches the page header (as mentioned in meeting).

**Comment by jasalisbury** [comment]
> We are calling them "non-recipients"? as in "recipient or non-recipient"?

Yeah I agree it seems a little strange. But yes I believe that is what was decided.

**Review by thewatermethod — APPROVED** [review]

**Comment by kryswisnaskas** [comment]
Looks good. Everything works as advertised. 👍 

(waiting for Patrice's final word on the non-recipient)

**Review by GarrettEHill — APPROVED** [review]

**Review by GarrettEHill — APPROVED** [review]

**Review by kryswisnaskas — CHANGES_REQUESTED** [review]
The wording looks good! 👍

**Review by kryswisnaskas — APPROVED** [review]

File: frontend/src/pages/ApprovedActivityReport/index.js [hunk 1]
```diff
@@ -178,7 +178,7 @@ export default function ApprovedActivityReport({ match, user }) {
         }
 
         // first table
-        let recipientType = data.activityRecipients[0].grantId ? 'Grantee' : 'Non-grantee';
+        let recipientType = data.activityRecipients[0].grantId ? 'Recipient' : 'Non-recipient';
```
Comment (thewatermethod) [inline frontend/src/pages/ApprovedActivityReport/index.js:181]: poking around the backend changes, noticed this

**Pull request merged**